core-web/libs/portlets/dot-ema/src/lib/shared/models.ts

```diff
@@ -3,6 +3,7 @@ export interface AddContentletPayload {
     container?: Container;
     pageContainers: Container[];
     pageID: string;
+    personaTag?: string;
```

Comment:
I made it not required because the default persona doesn't need it in the payload

Reply:
Why we just don't send it all the time instead of removing and adding the query param?

Reply:
Because we don't send the `personaTag` as queryParam, we need to modify the payload from this: 

```javascript
[
    {
        "identifier": "//demo.dotcms.com/application/containers/default/",
        "uuid": "10",
        "contentletsId": [
            "c151d3f0-9572-4bcc-8c8f-00c9da9758e0"
        ]
    },
    {
        "identifier": "//demo.dotcms.com/application/containers/default/",
        "uuid": "1",
        "contentletsId": [
            "df591adf-10fe-461a-a12e-f847df0fd2fb"
        ]
    },
    {
        "identifier": "//demo.dotcms.com/application/containers/default/",
        "uuid": "2",
        "contentletsId": []
    },
    {
        "identifier": "//demo.dotcms.com/application/containers/default/",
        "uuid": "3",
        "contentletsId": [
            "6bbead53-3908-471c-966d-9e76c5dd9eba"
        ]
    },
]
```

To this: 

```javascript
[
  {
    "identifier": "//demo.dotcms.com/application/containers/default/",
    "uuid": "10",
    "contentletsId": [
      "c151d3f0-9572-4bcc-8c8f-00c9da9758e0"
    ],
    "personaTag": "OceanEnthusiast"
  },
  {
    "identifier": "//demo.dotcms.com/application/containers/default/",
    "uuid": "1",
    "contentletsId": [
      "df591adf-10fe-461a-a12e-f847df0fd2fb"
    ],
    "personaTag": "OceanEnthusiast"
  },
  {
    "identifier": "//demo.dotcms.com/application/containers/default/",
    "uuid": "2",
    "contentletsId": [],
    "personaTag": "OceanEnthusiast"
  },
]
```

For personas, we send the personaTag, for the default we don't send anything. So the existence of the personaTag serves as a flag of how are we going to map the content of the payload

Reply:
And also, when we fetch the page as default persona, we don't get a `persona` property on `viewAs`